Marlin/src/HAL/HAL_LPC1768/arduino.cpp

```diff
@@ -85,6 +85,7 @@ extern "C" void delay(int msec) {
 
 // IO functions
 // As defined by Arduino INPUT(0x0), OUPUT(0x1), INPUT_PULLUP(0x2)
+#define OUPUT  0x1
```

Comment:
I thought that was strange.

The compiler gave me an error saying OUTPUT wasn't defined so I threw that in.

Arduino.cpp includes HAL.h which includes Arduino.h which has the definition of OUTPUT.  

I'll investigate further.